src/main.rs

```diff
@@ -111,8 +117,8 @@ struct SemverChecks {
     #[arg(long, global = true, exclusive = true)]
     list: bool,
 
-    #[command(flatten)]
-    verbosity: clap_verbosity_flag::Verbosity<clap_verbosity_flag::InfoLevel>,
```

Comment:
I assume we don't need `verbosity` anymore because `CheckRelease` already handles it?

Reply:
Correct, since we are flattening CheckRelease, we have the same verbosity flag available from the same places (with subcommand and not subcommand). Clap was actually the thing that highlighted it thanks to the "duplicated option" error.